doc/source/user/building.rst

```diff
@@ -137,7 +137,7 @@ Additional compiler flags can be supplied by setting the ``OPT``,
 Building with ATLAS support
 ---------------------------
 
-Ubuntu 
+Ubuntu
```

Comment:
This change is orthogonal to the PR. However, my editor auto-saves by stripping trailing whitespace and I figured it was worth staging the change for general hygiene. I'm happy to revert this line though if desired.

Reply:
No problem.